matching-notification-resources/src/main/resources/resources/uicomponents/matchingNotification/matchesTable.js

```diff
@@ -1007,21 +1065,123 @@ var PhenoTips = (function (PhenoTips) {
             }
             td += '<div name="notification-email-long">' + email + '</div>';
         }
-        var shortEmail = (emails.length > 0) ? (emails[0].substring(0, 9) + "...") :  "";
+        var shortEmail = (patient.emails.length > 0) ? (patient.emails[0].substring(0, 9) + "...") :  "";
         td += '<div name="notification-email-short">' + shortEmail + '</div>';
 
         //if logged as admin - add notification checkbox for local PC patient email contact but not for self (not for patients owned by admin)
-        if (this._isAdmin && serverId == '' && emails.length > 0) {
-            td += '<span class="fa fa-envelope" title="' + this._NOTIFY + '"></span> <input type="checkbox" class="notify" data-matchid="' + matchId + '" data-patientid="'+ patientId +'" data-emails="'+ emails.toString() +'">';
+        if (this._isAdmin && patient.serverId == '' && patient.emails.length > 0) {
+            td += '<span class="fa fa-envelope" title="' + this._NOTIFY + '"></span> <input type="checkbox" class="notify" data-matchid="' + matchId + '" data-patientid="'+ patient.patientId +'" data-emails="'+ patient.emails.toString() +'">';
         }
         td += '</td>';
         return td;
     },
```

Comment:
Note sure where in the code the change should be, but when a match is contacted outside of PC, the button "contact" is greyed out, but the history icon is also grey, which is a bit strange - greyed out button means it was contacted, but supposedly there is still no history

I would also make the "on hover" CSS for the history button more noticeable, it is absolutely not clear that the greyed out button can still be pressed (well, the cursor does change shape, but still)

Also I would add a bit more space between the button and the history icon, and decrease space on the right